app/react/Settings/components/NavlinkForm.js

```diff
@@ -6,11 +6,21 @@ import { findDOMNode } from 'react-dom';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 
-import { editLink } from 'app/Settings/actions/uiActions';
-import { removeLink } from 'app/Settings/actions/navlinksActions';
-import ShowIf from 'app/App/ShowIf';
+import { removeLink, addGroupLink, removeGroupLink } from 'app/Settings/actions/navlinksActions';
+
+// import ShowIf from 'app/App/ShowIf';
 import { Icon } from 'UI';
+import ShowIf from 'app/App/ShowIf';
+import './styles/menu.scss';
```

Comment:
Is it necessary to import and add scss/css directly into the component?

Reply:
Removed